job-dsls/jobs/kie/master/communityRelease_pipeline.groovy

```diff
@@ -765,7 +757,7 @@ matrixJob("${folderPath}/community-release-${baseBranch}-kieServerMatrix") {
     description("This job: <br> - Runs the KIE Server integration tests on mutiple supported containers and JDKs <br> IMPORTANT: Created automatically by Jenkins job DSL plugin. Do not edit manually! The changes will get lost next time the job is generated. ")
 
     // Label which specifies which nodes this job can run on.
-    label("master")
+    label('kie-rhel7&&kie-mem8g&&!master')
```

Comment:
This is for specification where the matrix parent job can run, I think you do not need to restrict it to `kie-mem8g`.